pipettes/firmware/stm32l5xx_it.c

```diff
@@ -124,6 +124,17 @@ void DebugMon_Handler(void)
 /*  file (startup_stm32l5xx.s).                                               */
 /******************************************************************************/
 
+void EXTI8_IRQHandler(void) {
+    HAL_GPIO_EXTI_IRQHandler(GPIO_PIN_8);
+}
+
+void EXTI9_IRQHandler(void) {
```

Comment:
We should probably conditionally initialize these depending on which pipette is loaded in.